templates/repo/diff/comment_form.tmpl

```diff
@@ -9,7 +9,7 @@
 		<input type="hidden" name="diff_start_cid">
 		<input type="hidden" name="diff_end_cid">
 		<input type="hidden" name="diff_base_cid">
-		<div class="ui top tabular menu" {{if not $.hidden}}onload="assignMenuAttributes(this)" {{end}}data-write="write" data-preview="preview">
```

Comment:
`assignMenuAttributes` becomes unused with this, should it be deleted?

Reply:
Nope, there are other callers.

<img width="871" alt="image" src="https://user-images.githubusercontent.com/2114189/141880571-1d61acd9-4d90-4cf0-a135-c0058d9e66b9.png">